packages/caip/src/adapters/coincap/index.ts

```diff
@@ -4,7 +4,7 @@ import toLower from 'lodash/toLower'
 import * as adapters from './generated'
 
 export const baseUrl = 'https://rest.coincap.io/v3'
-const apiKey = process.env.COINCAP_API_KEY || ''
+const apiKey = process.env.COINCAP_API_KEY || process.env.VITE_COINCAP_API_KEY || ''
```

Comment:
`yarn generate:all` is failing with coincap failures - we can probably remove left side but meh, not the point of this PR